### PR TITLE
instructions and fix

### PR DESCRIPTION
**Pull request by jburel**
This PR adds a README with instruction and fixes a path error
I have also added ``minio`` to the environment so it is fully self-contained

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -19,6 +20,7 @@ dependencies:
   - raw2ometiff
   - go-mc
   - pip
+  - minio
```
Comment (joshmoore) [inline environment.yml:23]: Your intent is to install minio outside of docker?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Create a conda environment using [environment.yml](environment.yml) file.
+```
+ conda env create -f
+```
+Activate the conda environment 
+```
+conda activate ngff-latency-benchmarking
+```
+
+Run the [setup.sh](setup.sh) script to generate the various imaging testing files from an IMARIS file.
```
Comment (joshmoore) [inline README.md:26]: If you change this to:

```
cd data
../setup.sh
```
then I think several of the other changes can be rolled back.

**Review by joshmoore — COMMENTED** [review]
One question and one change suggestion. Let me know if you want me to add to this PR.

File: environment.yml (hunk 1 above)
Comment (jburel) [reply environment.yml:23]: install minio in the conda environment as per the instruction.

Comment (joshmoore) [reply environment.yml:23]: > install minio in the conda environment as per the instruction.

Sorry, I still don't understand. Isn't that in the docker?

File: setup.sh [hunk 3]
```diff
@@ -13,4 +13,4 @@ mc policy set public benchmark/data
 mc cp -r *.ims benchmark/data/
 mc cp -r *.tiff benchmark/data/
 mc cp -r *.zarr benchmark/data/
-mc cp -r 1-byte benchmark/data/
+mc cp -r data/1-byte benchmark/data/
```
Comment (joshmoore) [inline setup.sh:16]: I think this needs rolling back as well.
```suggestion
mc cp -r 1-byte benchmark/data/
```

File: environment.yml (hunk 1 above)
Comment (jburel) [reply environment.yml]: sorry yes, I had something not correctly set up in the first instance. I started again from a clean sheet and it works nicely now

File: setup.sh (hunk 3 above)
Comment (jburel) [reply setup.sh]: I already pushed a commit for that change I did not see the suggestion

File: environment.yml [hunk 4]
```diff
@@ -1,3 +1,4 @@
+name: ngff-latency-benchmarking
```
Comment (joshmoore) [inline environment.yml:1]: I'll add for now, but the only downside of including `name:` is that then it's hard to make two versions locally.

Comment (jburel) [reply environment.yml:1]: I will roll it back and adjust command accordingly

**Pull request merged**